### PR TITLE
[enqueue] Add manifest for enqueue/pheanstalk  pkg

### PR DESCRIPTION
**Pull request by makasim**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: enqueue/pheanstalk/0.5/manifest.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "env": {
+        "ENQUEUE_DSN": "beanstalk://"
```
Comment (ASKozienko) [inline enqueue/pheanstalk/0.5/manifest.json]: LGTM

**Comment by makasim** [comment]
@fabpot how to get such PRs merged? Do I need always ping someone with a write access to this repo?

**Comment by fabpot** [comment]
@makasim We need at least one person who approves this pull request, someone from the organization of the bundle.

**Comment by makasim** [comment]
@fabpot 

@ASKozienko has approved it and he is the owner of php-enqueue org. I still does not see a merge button.

**Comment by fabpot** [comment]
@makasim Indeed, I will investigate.

**Comment by stof** [comment]
@fabpot looks like there is an issue about the way @symfony-bot detects approvals then.

**Comment by fabpot** [comment]
@stof not at all, that's a cache issue for "old" PRs.

**Pull request closed without merging**